helpers/constants.py

```diff
@@ -24,3 +24,38 @@
 # Skip anything other than metric docs in docs dir
 docs_to_skip = set(["readme.md", "docs-temp.md"])
 rtd_base_url = "https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations."
+
+monitor_doc_base_url = "https://docs.signalfx.com/en/latest/integrations/agent/monitors/"
+
+github_to_doc_map = {
+    "https://github.com/signalfx/integrations/tree/master/signalfx-agent": "https://docs.signalfx.com/en/latest/integrations/agent/index.html",
```

Comment:
Do we really have to hardcode all of these links?

Reply:
@keitwb Few of them are inevitable but I adopted some pattern match for the rest of it.